plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1548,4 +1549,75 @@ public static Object[][] batchSizeAndTotalNumberOfRowsToInsertDataProvider()
                 {10, 52}, // number of rows > n * batch size
         };
     }
+
+    // test polymorphic table function
```

Comment:
comments adorning a number of methods together are not recommended.
they tend to float away from methods being described, new methods may be aded in between, or comment may be left over after code moved/removed. See https://github.com/trinodb/trino/pull/11930 for example.

just remove the comment